model-server/src/test/kotlin/org/modelix/model/server/handlers/ModelReplicationServerTest.kt

```diff
@@ -113,6 +117,81 @@ class ModelReplicationServerTest {
         }
     }
 
+    @Test
+    fun `responds with 404 when deleting a branch from a non-existent repository`() {
+        val url = "http://localhost/v2"
+        val storeClient = InMemoryStoreClient()
+        val modelClient = LocalModelClient(storeClient)
+        val repositoriesManager = RepositoriesManager(modelClient)
+        val modelReplicationServer = ModelReplicationServer(repositoriesManager, modelClient, InMemoryModels())
+
+        runWithTestModelServer(modelReplicationServer) {
+            val response = client.delete {
```

Comment:
Should we directly add a `deleteBranch` method to `ModelClientV2`?

Reply:
it's added as a second commit now